src/electron/platform/android/appium-service-configurator.ts

```diff
@@ -2,24 +2,19 @@
 // Licensed under the MIT License.
 
 import ADB from 'appium-adb';
-import { AndroidServiceApkLocator } from 'electron/platform/android/android-service-apk-locator';
 import {
     AndroidServiceConfigurator,
     DeviceInfo,
     PackageInfo,
-    PermissionInfo,
 } from 'electron/platform/android/android-service-configurator';
 import { DictionaryStringTo } from 'types/common-types';
 
 type AdbDevice = {
     udid: string;
 };
 
-const servicePackageName: string = 'com.microsoft.accessibilityinsightsforandroidservice';
-
-export const defaultAdbPortNumber: number = 62442;
 export class AppiumServiceConfigurator implements AndroidServiceConfigurator {
```

Comment:
I think this should probably be renamed since it no longer contains any logic related to configuring our service, specifically; maybe `AdbAdapter` or `AppiumAdbAdapter`?